src/Features/Core/Portable/EncapsulateField/AbstractEncapsulateFieldService.cs

```diff
@@ -358,8 +360,9 @@ protected string GeneratePropertyName(string fieldName)
                 baseName = fieldName;
             }
 
-            // Make uppercase the first letter
-            return char.ToUpper(baseName[0]).ToString() + baseName.Substring(1);
+            // Make the first character upper case using the "en-US" culture
+            var firstCharacter = _englishUSCulture.TextInfo.ToUpper(baseName[0]);
```

Comment:
Can you add a comment and point to the issue on github around why this was done?